src/astarte-client/client.ts

```diff
@@ -447,20 +448,38 @@ class AstarteClient {
   }
 
   async getBlocks(): Promise<AstarteBlock[]> {
+    const mockedBlocks = definitions.blocks as AstarteBlockDTO[];
```

Comment:
those are not mocked blocks. maybe static blocks?
preconfigured blocks maybe?

Reply:
My bad, thanks for the heads up. I went with `staticBlocks`